airflow/jobs/scheduler_job.py

```diff
@@ -1275,10 +1275,13 @@ def processor_factory(file_path):
         # so the scheduler job and DAG parser don't access the DB at the same time.
         async_mode = not self.using_sqlite
 
+        processor_timeout_seconds = conf.getint('core', 'dagbag_import_timeout')
```

Comment:
IMO its good to have a fallback value here

Reply:
This is already defined in ./airflow/config_templates/default_airflow.cfg so it's not possible to not have a value for this set.